Makefile

```diff
@@ -1,6 +1,16 @@
 CORE = rel/epoch/bin/epoch
 VER := $(shell cat VERSION)
 
+CT_TEST_FLAGS =
+EUNIT_TEST_FLAGS =
+ifdef SUITE
+CT_TEST_FLAGS += --suite=$(SUITE)_SUITE
```

Comment:
It would be sweet to also support `--group` (important for `aecore_sync`)

Reply:
Fixed.